### DISCUSSION TITLE
Unable to extract Stress-Strain numerical value from HDF5 output file.

### DISCUSSION THREAD

**Discussion by st4rkman** [damask-multiphysics/DAMASK#23] [Q&A] [answered]
I ran the DAMASK simulation and got the dataset file in HDF5 format. Using the same file, I was able to extract .vti files and plot them on paraview, and later visualize the results for each increment, perfectly fine. 
Now, I need some Numerical data from the same HDF5 format (stress-strain values) but when I open the HDF5 file using HDFviewer it shows nothing, empty looking HDFviewer screen pops up. Hence I am unable to get stress-strain numerical values for every increment. Please help me with it.
![hdfviewer](https://github.com/user-attachments/assets/8ea33f86-3fdc-4836-9b6d-71a5ec52d978)

**Comment by MarDiehl** [comment, marked as answer]
I looks like the version of your HDF5view is too old. But anyways, HDFview is not really suitable for serious post-processing. For plotting average response, https://damask.mpie.de/documentation/examples/r-value_Marc.html is a good example

**Reply by st4rkman** [reply]
Thank you @MarDiehl for your response. I checked it with the new HDFViewer and it is working fine. However, I faced a problem picking out stress and strain values using Python. I took the reference of the link you provided and the video tutorial named: Data Analysis: Using Matplotlib and Pandas. 
There is some error while using the "r.get('sigma').values()" (though the screenshot has strain as an argument, it shows the same error for sigma as well) Python command when I am trying to extract the data despite following the video tutorial step by step. I'll attach the screenshot.
Please help me with it if I am making any mistakes.
![pythondamask](https://github.com/user-attachments/assets/269a8177-2f00-4ae5-b1c4-1570d28a1052)

**Reply by MarDiehl** [reply]
pls. have a look at https://damask.mpie.de/support/helpdesk for instructions when asking for help. Your screenshot is not helpful.

**Reply by st4rkman** [reply]
Okay. Thankyou @MarDiehl.
